cmd/internal/server/handlers/configuration_form.go

```diff
@@ -26,7 +27,7 @@ func (ConfigurationForm) Handle(ctx context.Context, _ *fivetransdk.Configuratio
 				Type: &fivetransdk.FormField_TextField{
 					TextField: fivetransdk.TextField_PlainText,
 				},
-				Required: true,
+				Required: &required,
```

Comment:
They might have some kind of Ptr func to make this easier but it's totally fine as is.

Reply:
Talked on Slack and checked - they do not have a Ptr function I think! Will keep in mind for future improvements 😄